DESCRIPTION

```diff
@@ -54,11 +54,12 @@ Suggests:
     knitr,
     rmarkdown,
     spelling,
-    testthat (>= 2.1.0)
+    testthat (>= 3.0.0)
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.1
 Language: en-US
 Encoding: UTF-8
 URL: https://jolars.github.io/SLOPE/, https://github.com/jolars/SLOPE
 BugReports: https://github.com/jolars/SLOPE/issues
 VignetteBuilder: knitr
+Config/testthat/edition: 3
```

Comment:
Testthat 3 has backwards-incompatible modifications; are you sure the tests still work?